x/evm/keeper/querier.go

```diff
@@ -88,6 +95,123 @@ func NewQuerier(k types.BaseKeeper, s types.Signer, n types.Nexus) sdk.Querier {
 	}
 }
 
+// QueryPendingCommands returns the pending commands for a gateway
+func QueryPendingCommands(ctx sdk.Context, keeper types.ChainKeeper, n types.Nexus) ([]byte, error) {
+	var resp types.QueryPendingCommandsResponse
+	commands := keeper.GetPendingCommands(ctx)
+
+	for _, cmd := range commands {
+		cmdResp, err := GetCommandResponse(ctx, keeper.GetName(), n, cmd)
+		if err != nil {
+			return nil, sdkerrors.Wrap(types.ErrEVM, err.Error())
+		}
+
+		resp.Commands = append(resp.Commands, cmdResp)
+	}
+
+	return resp.Marshal()
+}
+
+func queryCommand(ctx sdk.Context, keeper types.ChainKeeper, n types.Nexus, id string) ([]byte, error) {
+	cmdID, err := types.HexToCommandID(id)
+	if err != nil {
+		return nil, sdkerrors.Wrap(types.ErrEVM, err.Error())
+	}
+
+	cmd, ok := keeper.GetCommand(ctx, cmdID)
+	if !ok {
+		return nil, sdkerrors.Wrap(types.ErrEVM, fmt.Sprintf("could not find command '%s'", cmd.ID.Hex()))
+	}
+
+	resp, err := GetCommandResponse(ctx, keeper.GetName(), n, cmd)
+	if err != nil {
+		return nil, sdkerrors.Wrap(types.ErrEVM, err.Error())
+	}
+
+	return resp.Marshal()
+}
+
+// GetCommandResponse converts a Command into a CommandResponse type
+func GetCommandResponse(ctx sdk.Context, chainName string, n types.Nexus, cmd types.Command) (types.QueryCommandResponse, error) {
+	params := make(map[string]string)
+
+	switch cmd.Command {
+	case types.AxelarGatewayCommandDeployToken:
+		name, symbol, decs, cap, err := types.DecodeDeployTokenParams(cmd.Params)
+		if err != nil {
+			return types.QueryCommandResponse{}, err
+		}
+
+		params["token_name"] = name
```

Comment:
```suggestion
		params["name"] = name
```
these param names should reflect what's the the contracts.